covid_graphs/setup.py

```diff
@@ -17,9 +19,13 @@
         "scipy",
     ],
     dependency_links=[],
-    include_package_data=True,
     python_requires=">=3.7",
     packages=find_namespace_packages(),
-    scripts=["covid_graphs/scripts/prepare_data.py", "covid_graphs/scripts/scatter_plot.py", "covid_graphs/scripts/server.py"],
-    package_data={"covid_graphs": ["covid_graphs/scripts/index.html"]},
+    scripts=["covid_graphs/scripts/prepare_data.py", "covid_graphs/scripts/scatter_plot.py"],
+    entry_points={
+        "console_scripts": [
+            "covid_graphs.run_server=covid_graphs.scripts.server:run_server"
+        ]
+    },
+    package_data={"": ["*.html"]},
```

Comment:
This line was the tricky part. The syntax is
```
package_data={package_name: [glob_pattern_to_include_1, ...]}
```
It does not work though when I specify e.g. `{"covid_graphs": ["*.html"]}`, I think it might be because we use namespace packages (no `__init__.py` files).